packages/grant-explorer/src/hooks/matchingEstimate.ts

```diff
@@ -0,0 +1,63 @@
+import useSWR from "swr";
+import { Address } from "viem";
+import { ChainId } from "common";
+
+export type MatchingEstimateResult = {
+  totalReceived: string;
+  contributionsCount: string;
+  sumOfSqrt: string;
+  capOverflow: string;
+  matchedWithoutCap: string;
+  matched: string;
+  difference: bigint;
+  differenceInUSD: number;
+  roundId: string;
+  chainId: number;
+  recipient: string;
```

Comment:
When values are of a type that doesn't tell much, like `duration: number`, Joel would say to shift that task to the name, like `durationMs: number`.

Reply:
Which property are you referring to here?

Reply:
Recipient?

Reply:
Can't make specific suggestions though as I'm not sure I can guess the right meaning for each, but as probably-incorrect-examples: `recipient` → `recipientAddress`, `matched` → `matchedAmount`, `capOverflow`→ `capOverflowAmount`, `difference` → `priceDifference`.

Reply:
I see what you mean - I'm gonna drop a TODO to fix this once we have unified types between the indexer and frontend apps.